src/database/client/core/sync.ts

```diff
@@ -48,108 +60,182 @@ class DataSync {
   };
 
   connect = async (params: StartDataSyncParams) => {
-    const {
-      channel,
-      onSyncEvent,
-      onSyncStatusChange,
-      user,
-      onAwarenessChange,
-      signaling = 'wss://y-webrtc-signaling.lobehub.com',
-    } = params;
+    const { channel, onSyncEvent, onSyncStatusChange, user, onAwarenessChange } = params;
     // ====== 1. init yjs doc ====== //
 
     await this.initYDoc();
 
     this.logger('[YJS] start to listen sync event...');
     this.initYjsObserve(onSyncEvent, onSyncStatusChange);
 
-    // ====== 2. init webrtc provider ====== //
-    this.logger(`[WebRTC] init provider... room: ${channel.name}`);
-    const { WebrtcProvider } = await import('y-webrtc');
+    if (channel.webrtc.enabled) {
```

Comment:
I think there can be two methods: `connectWithWebRTC` and `connectWithLiveBlocks` ,

move the implement into these method and make the `connect` as entry.